integration_tests/macro/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+use proc_macro::TokenStream;
+use quote::{format_ident, quote};
+
+#[proc_macro_attribute]
+pub fn exported_test(_attr: TokenStream, item: TokenStream) -> TokenStream {
```

Comment:
Please add some documentation here, both what it does, and the intention behind it.